### PR TITLE
Pick up Node Function and Yarn Buildpack new releases

### PR DESCRIPTION
**Pull request by fbiville**

### CODE REVIEW

File: builder.toml.tpl [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
+  { id = "io.projectriff.node",          uri = "https://storage.googleapis.com/projectriff/node-function-buildpack/io.projectriff.node-0.3.0.tgz" },
 
   { id = "org.cloudfoundry.buildsystem", uri = "https://storage.googleapis.com/cnb-buildpacks/build-system-cnb/org.cloudfoundry.buildsystem-{{ go mod download -json | jq -r 'select(.Path == "github.com/cloudfoundry/build-system-cnb").Version' | sed -e 's/^v//g' }}.tgz" },
   { id = "org.cloudfoundry.node-engine", uri = "https://github.com/cloudfoundry/node-engine-cnb/releases/download/{{ go mod download -json | jq -r 'select(.Path == "github.com/cloudfoundry/node-engine-cnb").Version' }}/node-engine-cnb-{{ go mod download -json | jq -r 'select(.Path == "github.com/cloudfoundry/node-engine-cnb").Version' | sed -e 's/^v//g' }}.tgz" },
-  { id = "org.cloudfoundry.yarn-install",        uri = "https://github.com/cloudfoundry/yarn-install-cnb/releases/download/v0.1.14/yarn-install-cnb-0.1.14.tgz" },
+  { id = "org.cloudfoundry.yarn-install",        uri = "https://github.com/cloudfoundry/yarn-install-cnb/releases/download/v0.1.16/yarn-install-cnb-0.1.16.tgz" },
```
Comment (scothis) [inline builder.toml.tpl:9]: how about we template this like the other cf buildpacks

Comment (fbiville) [reply builder.toml.tpl:9]: ah, it was only templated in the `.tpl.tpl`, not in the `.tpl`

Comment (fbiville) [reply builder.toml.tpl]: Where is the matryoshka emoji when we need it? :(

**Comment by fbiville** [comment]
> I'd expect to see an update to the builder.toml.tpl.tpl file

Yep, I missed the `tpl_escape` call when I initially introduced the Yarn buildpack.
Now fixed.

**Pull request merged**